<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recordplan</class>
 <widget class="QDialog" name="Recordplan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>812</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>812</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <widget class="QLabel" name="record_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>812</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/record_page.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="go_secpage">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>59</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="expense_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p &gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 14pt &quot;맑은 고딕&quot;;
	color: rgb(0,0,0);
	background-color: white;
	border: 2px solid rgb(0,0,0);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Expense</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="income_btn">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>290</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 14pt &quot;맑은 고딕&quot;;
	color: rgb(0,0,0);
	background-color: white;
	border: 2px solid rgb(0,0,0);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Income</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="money_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'맑은 고딕'; font-size:12pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(0,0,0);
background-color: rgb(255, 255, 255);
font: 700 14pt &quot;맑은 고딕&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>208</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 700 14pt &quot;맑은 고딕&quot;;</string>
    </property>
    <property name="text">
     <string>Add title</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="category_list">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>385</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>Meal</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Meal.png</normaloff>:/images/Meal.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cafe/Snack</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Snack.png</normaloff>:/images/Snack.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medicine</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Medicine.png</normaloff>:/images/Medicine.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Car</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Car.png</normaloff>:/images/Car.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transportation</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Transportation.png</normaloff>:/images/Transportation.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Education</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Education.png</normaloff>:/images/Education.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Living</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Living.png</normaloff>:/images/Living.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Travel</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Travel.png</normaloff>:/images/Travel.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Daily</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Daily.png</normaloff>:/images/Daily.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Exercise</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Exercise.png</normaloff>:/images/Exercise.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothes</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Clothes.png</normaloff>:/images/Clothes.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beauty</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Beauty.png</normaloff>:/images/Beauty.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>custom_category</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/Other.png</normaloff>:/images/Other.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>650</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 16pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border: 2px solid rgb(0,0,0);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="method_list">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>465</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>Card</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other thing</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="remove_money">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>136</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>543</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>1752</year>
      <month>10</month>
      <day>26</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="go_category_menu">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>385</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
